src/styles/main.scss

```diff
@@ -0,0 +1,103 @@
+.container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.product {
+  @include flex-column;
+
+  width: $product-full-width;
+  background-color: $background;
+  align-items: center;
+  text-align: center;
+  padding-block: 34px 0;
+  &-image {
+    width: $product-full-width * 0.8;
```

Comment:
This value should have fixed width in pixels without any calculations. Please check the mockup.

Reply:
The same for other elements.